api-doc/api-doc.yaml

```diff
@@ -0,0 +1,2280 @@
+openapi: 3.0.0
```

Comment:
@borkarsaish65 I amseeing some error in this document . please check if they can be addressed

Errors
Hide
 
Structural error at paths./v1/surveySubmissions/list.post.responses
should only have three-digit status codes, `default`, and vendor extensions (`x-*`) as properties
Jump to line 1778
Structural error at paths./v1/surveySubmissions/isAllowed/:surveySubmissionId.get.responses
should only have three-digit status codes, `default`, and vendor extensions (`x-*`) as properties
Jump to line 1854
Structural error at paths./v1/solutions/details/:solutionId.post.requestBody
should have required property 'content'
missingProperty: content
Jump to line 2274

Reply:
removed most of the errors